src/parser/Parser.ts

```diff
@@ -1364,6 +1368,38 @@ export class Parser {
         return new ThrowStatement(throwToken, expression);
     }
 
+    private dimStatement() {
+        const dim = this.advance();
+
+        //get the class name
```

Comment:
I think this is the wrong comment. There's nothing to do with class here.

Reply:
Removed